exchanges/okx/okx.go

```diff
@@ -3653,10 +3514,10 @@ func (ok *Okx) GetLiquidationOrders(ctx context.Context, arg *LiquidationOrderRe
 	default:
 		return nil, errEitherInstIDOrCcyIsRequired
 	}
-	if (arg.InstrumentType == okxInstTypeFutures || arg.InstrumentType == okxInstTypeSwap || arg.InstrumentType == okxInstTypeOption) && arg.Underlying != "" {
+	if arg.InstrumentType != okxInstTypeMargin && arg.Underlying != "" {
 		params.Set("uly", arg.Underlying)
 	}
-	if arg.InstrumentType == okxInstTypeFutures && (arg.Alias == "this_week" || arg.Alias == "next_week" || arg.Alias == "quarter" || arg.Alias == "next_quarter") {
+	if arg.InstrumentType == okxInstTypeFutures && arg.Alias != "" {
 		params.Set("alias", arg.Alias)
 	}
 	if ((arg.InstrumentType == okxInstTypeFutures || arg.InstrumentType == okxInstTypeSwap) && arg.Alias == "unfilled") || arg.Alias == "filled" {
```

Comment:
As mentioned before, this looks incorrect. You're trying to set the "alias" as "underlying" after verifying the alias. The docs also don't support any "underlying" types being used in the "alias" field, so I'm assuming this is a typo